californium-core/src/main/java/org/eclipse/californium/core/network/Exchange.java

```diff
@@ -466,8 +489,9 @@ public boolean isComplete() {
 	public void setComplete() {
 		this.complete = true;
 		ExchangeObserver obs = this.observer;
-		if (obs != null)
+		if (null != obs) {
```

Comment:
:-) my be I'm too influenced from MISRA.

Would the use of `final ExchangeObserver obs` be acceptable?